c2corg_ui/templates/helpers/edit.html

```diff
@@ -111,7 +111,19 @@ <h3 class="heading show-phone" translate>summary</h3>
     </div>
   </section>
 </%def>
-
+<%def name="show_cotometer_dialog(module, lang, other_langs)">\
```

Comment:
It's not complicated :)
Instead of defining your function twice (in 2 helpers files), you define it only once either in an existing helpers file or - if no existing helpers file is relevant - you may create a new one and move your function to it.
Then you import your function in the template using the following syntax:
```
<%namespace file="some/path/to/the/helperfile.html" import="show_cotometer_dialog"/>
```